src/main/java/net/sf/jabref/model/entry/BibEntry.java

```diff
@@ -194,7 +202,6 @@ public void setType(EntryType type) {
     /**
      * Returns the contents of the given field, or null if it is not set.
      */
-    @Deprecated //Use getFieldOptional instead
```

Comment:
https://github.com/JabRef/jabref/pull/1451/files#diff-044ee1a556d146810fd96d93f040505aR514
I cannot write the method like this
```java
public static String escapeValue(Optional<Object> obj) {
   String stringValue;
      if (obj.isPresent()) {
         if (obj.get() instanceof String) {
            stringValue = "'" + obj.get() + "'";
         } else {
            stringValue = String.valueOf(obj.get());
         }
      } else {
         stringValue = "NULL";
      }
      return stringValue;
}
```

... because `Optional<Object>` does not allow any assignment of `Optional<T>` where `T` is an arbitrary type. But we need the `escapeValue` method for `Integer` and other types.